Sources/Classes/core/Storage/MiniAppSecureStorage.swift

```diff
@@ -32,8 +32,9 @@ public class MiniAppSecureStorage: MiniAppSecureStorageDelegate {
     private static let storageName: String = "securestorage"
     static var storageFullName: String { return storageName + ".plist" }
 
-    public init(appId: String) {
+    public init(appId: String, storageMaxSizeInBytes: UInt64? = nil) {
         self.appId = appId
+        self.fileSizeLimit = storageMaxSizeInBytes ?? 2_000_000
```

Comment:
We should update this in documentation as well. Default size is supported by the SDK itself.
@corycaywood , I think we should update the same in storage documentation?